### PR TITLE
Issue/1575 omit already shared

### PR DESCRIPTION
**Pull request by walid-i**
When sharing omits users that already have access

### CODE REVIEW

File: packages/app/obojobo-repository/shared/components/people-search-dialog.jsx [hunk 1]
```diff
@@ -35,13 +53,28 @@ const PeopleSearchDialog = props => {
...
+						} else {
+							return (
+								<PeopleListItem key={p.id} isMe={p.id === props.currentUserId} {...p}>
+									<Button className="select-button" onClick={() => onSelectPerson(p)}>
+										Select
+									</Button>
+								</PeopleListItem>
+							)
+						}
+					})} */}
```
Comment (FrenjaminBanklin) [inline packages/app/obojobo-repository/shared/components/people-search-dialog.jsx:76]: Remove unused commented code.

**Review by FrenjaminBanklin — CHANGES_REQUESTED** [review]
Looks good to me.

Just remove the large commented block from `packages/app/obojobo-repository/shared/components/people-search-dialog.jsx` and it should be all set.

**Comment by RachelDau** [comment]
I merged in dev 30 except four style files which have lint errors. These lint errors appear to be already in dev30, so if I alter the files in order to commit them then they will continue to shop up on this PR diff. I was not sure if that was ideal so I didn't not commit those files yet. They show up in the diff right now without the dev30 merge updates.

**Comment by RachelDau** [comment]
merged with Dev 30 and lint style updates made in 4 files. These four files more-info-box.scss, objective-list-view.scss, search.scss, and objective-item.scss are seperate from this PR, but needed their styles to be updated to be merged in. I also removed the commented code requested to be removed above.

**Review by FrenjaminBanklin — CHANGES_REQUESTED** [review]
Code isn't building to test, see below.

**Review by FrenjaminBanklin — APPROVED** [review]
Looks like this is working properly and performing as advertised.

**Pull request merged**